inlong-tubemq/conf/master.ini

```diff
@@ -19,7 +19,7 @@
 ; host address of master, required; must be configured at network card, enabled
 hostName=127.0.0.1
 ; port that master listens to, optional; default is 8715
-port=8000
+port=8715
```

Comment:
Does the modification to 8715 have any special meaning?

Reply:
As shown in the comment in Line 21, the default port is 8715. Besides, the default port in `MasterConfig` is 8715, too.
So maybe it's better to use 8715 here, so it won't confuse users.

Reply:
Ok